ansible/roles/docker/ecr/tasks/main.yml

```diff
@@ -147,7 +147,7 @@
             raw: 'apt-get -y update && apt-get purge -y python3-openssl'
             register: output
             when:
-              - ansible_failed_result['module_stderr'] | regex_search("module 'lib' has no attribute 'X509_V_FLAG_.*")
+              - ansible_failed_result['stderr'] | regex_search("module 'lib' has no attribute 'X509_V_FLAG_.*")
```

Comment:
We've seen this is not the solution.
@carebare47 was going to try `ansible_failed_result.msg`

Reply:
tested, ansible_failed_result['msg'] and ansible_failed_result.msg both work. Gone with `ansible_failed_result.msg` for consistency

Reply:
60b018319fb12d444d04998a00acbf443f114d5e